Maestro/src/Microsoft.DotNet.Maestro.WebApi/Models/ModifiedFileModel.cs

```diff
@@ -42,7 +46,8 @@ public class ModifiedFileModel
                 RepoName = repoFullName,
                 BranchName = branchName,
                 FullPath = "https://github.com/" + string.Join("/", repoFullName, "blob", branchName, fullPath),
-                CommitId = commit.id
+                CommitId = commit.id,
+                CommitList = string.Join(";", Commits.ConvertAll( t => t.id ))
```

Comment:
I would have expected this to be `Commits.Select(c => c.id)` .

Differences:

1. Select instead of ConvertAll.  (Select works on more collection types, not just List)
2. the param `c` instead of `t`.  I'm not sure what `t` stands for.